code/modules/mob/living/damage_procs.dm

```diff
@@ -197,9 +200,12 @@
 /mob/living/proc/setOxyLoss(amount, updating_health = TRUE, forced = FALSE, required_biotype, required_respiration_type = ALL)
 	if(!forced && (status_flags & GODMODE))
 		return
-	if(iscarbon(src))
-		var/obj/item/organ/internal/lungs/affected_lungs = getorganslot(ORGAN_SLOT_LUNGS)
-		if(!forced && !(affected_lungs?.respiration_type & required_respiration_type))
+	var/obj/item/organ/internal/lungs/affected_lungs = getorganslot(ORGAN_SLOT_LUNGS)
+	if(!affected_lungs)
+		if(!(mob_respiration_type & required_respiration_type))
+			return
+	else 
+		if(!forced && !(affected_lungs.respiration_type & required_respiration_type))
 			return
```

Comment:
Why don't we check `!forced` on the top example (where if we don't have any `affected_lungs`, like what you did in `adjustOxyLoss`? Either way, might be apt to just follow the instructions in the last review.